### PR TITLE
Add story about mysterious text

### PR DESCRIPTION
**Pull request by lpatino10**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2491)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/check-phone/check-phone.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+You get a notification that someone has texted you, but you don't 
+recognize the number. You open your messaging app, but you can't see
+a preview of the text. Do you:
+
+[Open the message](../open-message.md)
```
Comment (casey-collab) [inline english/check-phone/check-phone.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/check-phone/check-phone.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+[Delete the message string](../delete-message.md)
```
Comment (casey-collab) [inline english/check-phone/check-phone.md]: This link is also invalid.

File: english/check-phone/open-message/open-message.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+You open up the message, but nothing appears. Suddenly, the screen turns
```
Comment (casey-collab) [inline english/check-phone/open-message/open-message.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/check-phone/check-phone.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+You get a notification that someone has texted you, but you don't
```
Comment (casey-collab) [inline english/check-phone/check-phone.md]: This page is also unreachable.

File: english/check-phone/delete-message/delete-message.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+You press delete and the message string is gone. A few weeks later, you
```
Comment (casey-collab) [inline english/check-phone/delete-message/delete-message.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by lpatino10** [comment]
@casey-collab the links are fixed!

File: english/check-phone/check-phone.md (hunk 4 above)
Comment (casey-collab) [inline english/check-phone/check-phone.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/check-phone/check-phone.md (hunk 4 above)
Comment (cbuckey-uda) [inline english/check-phone/check-phone.md]: Sorry, Casey does not understand links that span two lines, as the link that leads to this page does. This page actually is reachable.

**Comment by cbuckey-uda** [comment]
Looks good to me! :phone:

**Pull request merged**